migration-engine/cli/src/commands.rs

```diff
@@ -24,10 +24,21 @@ impl Cli {
     }
 
     pub(crate) async fn run_inner(self) -> Result<String, ConnectorError> {
+        let datamodel = datasource_from_database_str(&self.datasource)?;
```

Comment:
that naming is confusing

Reply:
I agree. The good news is that when the CLI takes advantage of `create_database()` and `can_connect_to_database()` in the JSON-RPC API, this can be deleted.